server/controllers/find/courseOfferingsController.ts

```diff
@@ -13,9 +20,15 @@ export default class CourseOfferingsController {
     return async (req: Request, res: Response) => {
       TypeUtils.assertHasUser(req)
 
-      const course = await this.courseService.getCourseByOffering(req.user.token, req.params.courseOfferingId)
-      const courseOffering = await this.courseService.getOffering(req.user.token, req.params.courseOfferingId)
-      const organisation = await this.organisationService.getOrganisation(req.user.token, courseOffering.organisationId)
+      const [course, [courseOffering, organisation,]] = await Promise.all([
+        this.courseService.getCourseByOffering(req.user.token, req.params.courseOfferingId),
+        this.courseService.getOffering(req.user.token, req.params.courseOfferingId)
+          .then(async (courseOffering) => {
+            const organisation = await this.organisationService.getOrganisation(req.user.token, courseOffering.organisationId)
+            return [courseOffering, organisation,] as orgCourseOfferings
```

Comment:
If you have styling opinions can you please push for a linting change.

Reply:
A linting option would be nice, but there isn't the time to look into that right now. For now, it would be good to keep things roughly in line with the existing codebase

Reply:
I agree with the inline type and will change that.